packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2632,18 +2641,28 @@ export class ContainerRuntime
 	}
 
 	private isContainerMessageDirtyable({ type, contents }: OutboundContainerRuntimeMessage) {
-		// For legacy purposes, exclude the old built-in AgentScheduler from dirty consideration as a special-case.
-		// Ultimately we should have no special-cases from the ContainerRuntime's perspective.
-		if (type === ContainerMessageType.Attach) {
-			const attachMessage = contents as InboundAttachMessage;
-			if (attachMessage.id === agentSchedulerId) {
-				return false;
+		// Certain container runtime messages should not mark the container dirty such as the old build-in
```

Comment:
Should be "built-in"

```suggestion
		// Certain container runtime messages should not mark the container dirty such as the old built-in
```